jfxtras-controls/src/main/java/jfxtras/internal/scene/control/skin/CalendarTextFieldSkin.java

```diff
@@ -67,6 +67,13 @@
  */
 public class CalendarTextFieldSkin extends SkinBase<CalendarTextField>
 {
+    
```

Comment:
Sames argument about style applies here: if it is your code, you are free do it as you like. This is my code, and I use the pattern of placing variables below the method where they are initialized, in this case that is below createNodes.